core/frontend/src/PerModelCategoryVisibility.ts

```diff
@@ -103,27 +113,70 @@ class PerModelCategoryVisibilityOverrides extends SortedArray<PerModelCategoryVi
       return PerModelCategoryVisibility.Override.None;
   }
 
-  public setOverride(modelIds: Id64Arg, categoryIds: Id64Arg, override: PerModelCategoryVisibility.Override): void {
+  /**
+   *
+   * @param perModelCategoryVisibility array of model category visibility overrides
+   * @param iModel Optional param iModel. If no iModel is provided, then the iModel associated with the viewport (used to construct this class) is used.
+   * This optional iModel param is useful for apps which may show multiple iModels at once. Passing in an iModel ensures that the subcategories cache for the provided iModel
+   * is populated as opposed to the iModel associated with the viewport which may or may not be an empty iModel.
+   * @returns void promise
```

Comment:
"a promise that resolves once the overrides have been applied."

Reply:
changed